start-page/index.html

```diff
@@ -126,7 +126,10 @@
 		    <p>• Ohso Ltd for the OMG! Ubuntu logo</p>
 		    <p>• Unsplash for the Unsplash logo</p>
 		    <p>• Nohat for the Wikipedia Logo (concept by Paullusmagnus) <a href="https://creativecommons.org/licenses/by-sa/3.0/">(CC BY-SA 3.0)</a></p>
-		    <p>• <a href="https://github.com/feren-OS/feren-os.github.io/pulls?utf8=✓&q=is%3Apr+is%3Amerged">Everyone who has contributed improvements to this Start Page</a></p></div>
+				<p>• <a href="https://github.com/feren-OS/feren-os.github.io/pulls?utf8=✓&q=is%3Apr+is%3Amerged">Everyone who has contributed improvements to this Start Page</a></p></div>
```

Comment:
I'm assuming the odd tabbing here is just your code editor? Eh, nothing that'll ruin the PR (I don't think so anyway at least), but rather just an observation and something I'll edit when integrating this pull request into the frontend.

Reply:
Yeah, VS Code tabbed it a little weird, as my fonts are messed up